### DISCUSSION TITLE
New Feature: More than single park entrance type

### DISCUSSION THREAD

**Discussion by rosarian** [OpenRCT2/OpenRCT2#17031] [Feature requests] [answered]
The new file format permits a lot more peep spawning places and (presumably, I haven't seen this documented) park entrances.
However, only 1 entrance type is permitted.  It would be fun to have themed entrances for different areas of the park.

**Comment by sjoerddebruin** [comment, marked as answer]
Duplicate of #15750, has been added in #21909.
